backend/app/assets/javascripts/spree/backend/views/payment/payment_row.js.coffee

```diff
@@ -0,0 +1,24 @@
+Spree.Views.Payment.PaymentRow = Backbone.View.extend
```

Comment:
I think the file should be in javascript instead coffee.

From [backend README](https://github.com/solidusio/solidus/tree/master/backend):

>Though we have existing CoffeeScript files, any new files should be in javascript (ES5).

Reply:
Ah, you are right, good catch!

Reply:
Will do, thanks!